---
 layout: post
 title: TSt
 categories:
 - 科
 tags:
 - pro
---

<h1>The Protein-Folding Problem, 50 Years On<br>蛋白质折叠问题的 50 年历程</h1>

<div id="dbLayout">
The protein-folding problem was first posed about one half-century ago. The term refers to three broad questions: (i) What is the physical code by which an amino acid sequence dictates a protein's native structure? (ii) How can proteins fold so fast? (iii) Can we devise a computer algorithm to predict protein structures from their sequences? We review progress on these problems. In a few cases, computer simulations of the physical forces in chemically detailed models have now achieved the accurate folding of small proteins. We have learned that proteins fold rapidly because random thermal motions cause conformational changes leading energetically downhill toward the native structure, a principle that is captured in funnel-shaped energy landscapes. And thanks in part to the large Protein Data Bank of known structures, predicting protein structures is now farmore successful than was thought possible in the early days. What began as three questions of basic science one half-century ago has now grown into the full-fledged research field of protein physical science.

蛋白质折叠问题是在半个世纪前第一次提出来的。这一术语提到了三大问题:(1) 氨基酸序列决定蛋白质天然结构的物理密码是什么? (2)蛋白质是怎样迅速折叠 的?(3) 我们能设计一种计算机算法，让这种算法根据蛋白质的氨基酸序列就能 预测蛋白质的结构吗?我们回顾在这些问题上所取得的进展。只在少数情况下, 对于精细的化学模型中的物理外力，计算机模拟能够实现小型蛋白的正确折叠。我们了解到蛋白质迅速折叠是因为随机热运动使得构象发生改变，而根据漏斗状 的能量图景（funnel-shaped energy landscapes）中所体现的原理，构象变化会使 蛋白质沿着下坡路回归到天然结构。还得感谢蛋白质数据库（PDB）中的已知结 构,现在预测蛋白质结构远比早期预想的要成功。半个世纪前，以这三个问题开 始的基础科学现如今已经发展为一个对蛋白质自然科学进行研究的的成熟领域。

Protein molecules embody a remarkable relationship between structure and function at the molecular level. Proteins perform many different functions in biochemistry. A protein's biological mechanism is determined by its three-dimensional (3D) native structure, which in turn is encoded in its 1D string of amino acid monomers.

蛋白质分子在分子水平上具体体现了蛋白质结构和其功能之间的显著关系。蛋白 质在生物化学中有许多不同的功能。蛋白质的三维(3 D)天然结构决定它的生物 学机制，而反过来，蛋白质的氨基酸单体的一维链又对它的天然机构进行编码。

This year marks the 50th anniversary of the 1962 Nobel Prize in Chemistry awarded to for their pioneering work in determining the structure of globular proteins (1–3). That work laid the foundation for structural biology, which interprets molecular level biological mechanisms in terms of the structures of proteins and other biomole cules. Their work also raised the question of how protein structures are explained by physical principles. Upon seeing the structure of myoglobin at 6 ? resolution (1), Kendrewet al.said,

1962 年诺贝尔化学奖授予给了 Max Perutz 和 John Kendrew，以表彰他们在确定 球状蛋白质的结构这个方面所做的开创性工作 (1 - 3)，而今年是其 50 周年纪念日。他们的工作奠定了结构生物学基础,也从蛋白质和其他生物分子的角度解释 了分子水平上的生物机制。他们的工作也提出这样的问题，怎样用物理原理来阐 明蛋白质的结构？如图 1 所示的是分辨率为 6 ? 时的肌红蛋白结构，如 Kendrewet al.所说，

Perhaps the most remarkable features of the molecule are its complexity and its lack of symmetry. The arrangement seems to be almost totally lacking in the kind of regularities which one instinctively anticipates, and it is more complicated than has been predicated by any theory of protein structure. Though the detailed principles of construction do not yet emerge, we may hope that they will do so at a later stage of the analysis.

或许这种分子最引人注目的特征就是其复杂性和缺乏对称性。这种安排似乎完全 缺少了人们本能会期望的那种规律,它比任何一种蛋白质结构理论所预测的还要 复杂。尽管还没有这个结构的详细原理，但我们希望在后期的分析阶段他们会做 这样的工作。

Fig 1
图 1.
In 1958, Kendrew and co-workers published the first structure of a globular protein: myoglobin at 6 ? resolution (1). Its puzzlingly complex structure lacked the expected symmetry and regularity and launched the protein-folding problem. [With permission from the Medical Research Council Laboratory of Molecular Biology]

1958 年，Kendrew 和他的同事公开了第一个球状蛋白质的结构：在 6 ? 分辨率下的肌红蛋白（1） 。它令人 莫名其妙的复杂结构缺乏对称性和规则性，而肌红蛋白也正式开启了蛋白质折叠问题。[经医学研究委员会 分子生物实验室同意]

The protein-folding problem came to be three main questions: (i) The physical folding code: How is the 3D native structure of a protein determined by the physicochemical properties that are encoded in its 1D amino-acid sequence? (ii)The folding mechanism: A polypeptide chain has an almost unfathomable number of possible conformations. How can proteins fold so fast? (iii)Predicting protein structures using computers: Can we devise a computer algorithm to predict a protein's native structure from its amino acid sequence? Such an algorithm might circumvent the time-consuming process of experimental protein structure determination and accelerate the discovery of protein structures and new drugs. Here, we give our perspective on these questions at the broad-brush level. More detailed reviews can be found elsewhere (4–8)。

蛋白质折叠问题主要形成了三个问题： (i) 物理折叠密码：蛋白质的一维氨基酸 序列的理化特性是怎样决定蛋白质的 3 维天然结构的？ (ii)折叠机制： 一个多 肽链可能的构象是不计其数的。蛋白质是如何折叠得这么快的？(iii) 使用计算机 来预测蛋白质结构： 我们能设计一种计算机算法，让这种算法根据蛋白质的氨 基酸序列就能预测蛋白质的结构吗? 这种算法可能会避开蛋白质结构测定实验 中费时的过程，从而加快蛋白质结构和新药物的发现。在这里，我们粗略的给出 了我们对这些问题的观点。更多细节请参考(4-8)。

### The Physical Code of Protein Folding
### 蛋白质折叠的物理密码

What forces drive a protein to its 3D folded structure? Much insight comes from the Protein Data Bank (PDB), a collection of now more than 80,000 protein structures at atomic detail (9). The following factors appear to contribute (10): (i) Hydrogen bonds. Protein structures are composed of a-helices and b-sheets, as was predicted by Linus Pauling on the basis of expected hydrogen bonding patterns (11). (ii) van der Waals interactions. The atoms within a folded protein are tightly packed, implying the importance of the same types of close-ranged interactions that govern the structures of liquids and solids. (iii) Backbone angle preferences. Like other types of polymers, protein molecules have preferred angles of neighboring backbone bond orientations. (iv) Electrostatic interactions. Some amino acids attract or repel because of negative and positive charges. (v) Hydrophobic interactions. Proteins ball up into well-packed folded states in which the hydrophobic (H) amino acids are predominantly located in the protein's core and the polar (P) amino acids are more commonly on the folded protein's surface. Theory and experiments indicate that folding is governed by a predominantly binary code based on interactions with surrounding water molecules: There are few ways a given protein sequence of H and P residues can configure to bury its hydrophobic amino acids optimally (12,13). (vi) Chain entropy. Opposing the folding process is a large loss in chain entropy as the protein collapses into its compact native state from its many open denatured configurations (12).

什么样的作用力促使蛋白形成了它的三维折叠结构？PDB 现已集合了 80,000 多 个原子级别的蛋白质结构，人们从 PDB 中得出很多见解 (9)。以下因素似乎促 成了蛋白质的三维折叠结构(10)： (i) 氢键。正如莱纳斯· 鲍林基于氢键预期模式 的预测一样，蛋白质结构是由 a-螺旋和 b-折叠组成(11)。(ii) 范德华力相互作用。折叠蛋白质中的原子都紧紧的挤在一起，这暗示了相同类型中紧密排列时相互作用的重要性，因为这种相互作用决定了液体和固体结构。(iii)骨架角度倾向性。像其他类型的聚合物一样，蛋白质分子也有角度倾向性：向相邻的骨架结合的方 向。(iv) 静电相互作用。一些氨基酸因为正电荷和负电荷而相互吸引或排斥。(v) 疏水作用。当疏水氨基酸(H)序列主要位于蛋白质核心，亲水氨基酸（P）普遍位 于折叠蛋白质的表面时，蛋白质团成精密的折叠状态。理论和实验表明折叠主要 是由二元码控制，而这二元码是基于周围水分子的相互作用：疏水残基和亲水残 基中的给定的蛋白质序列几乎没有办法通过配置最好地隐藏其疏水氨基酸 (12，13)。(六) 链熵。当蛋白质从它的多开放变性组态折叠回紧凑的天然状态时，阻碍折叠进程是链熵的巨大损失。

These physical forces are described approximately by―forcefields‖(14). Forcefields are models of potential energies that are used in computer simulations. They are widely applied to studies of protein equilibria and dynamics. In computer modeling, a protein molecule is put into an initial configuration, often random. Conformations change over the course of the simulation by repeatedly solving Newton's dynamical laws of motion for the atoms of the protein molecule and the solvent by using the forcefield energies. According to the laws of thermodynamics, systems tend toward their states of lowest free energy. Computational protein folding explores the process by which the protein proceeds through conformational states to states of lower free energies. As shown in Fig. 2, the thermodynamically stable states of 12 small protein structures can be reached fairly successfully by means of extensive molecular dynamics (MD) simulations in a bath of explicit water molecules (15). However, such successes, important as they are, are limited. So far,such modeling succeeds only on a limited set of small simple protein folds (16). And, it does not yet accurately predict protein stabilities or thermodynamic properties. Opportunities for the future include better forcefields, better models of the protein-water interactions, and faster ways to sample conformations, which are far too limited, even with today's most powerful computers.

这些物理外力都可以近似的由力场（forcefields）来描述 (14)。“力场”是指能 用于计算机模拟的能量模型。他们被广泛地应用于蛋白质平衡和动力学的研究。在计算机建模中，蛋白质分子常常是随机的放入一个初始组态中。构象在模拟过 程中发生改变是需要用能量力场不断求解蛋白质分子中的原子和溶剂的牛顿动 力学运动定律方程。根据热力学定律，系统的状态往往趋向于自由能最低的状态。蛋白质折叠计算探索了蛋白质从构象状态到最低自由能状态的过程。如图 2 所示，通过对溶剂中的水分子进行分子动力学模拟，12 个小型蛋白质结构可以相当成 功的转化为热力学上的稳定状态(15)。然而，这样成功尽管重要却也受到限制。到目前为止，这样的建模只在有限的小型简单蛋白质折叠过程中成功过(16)。并 且，它也还没有准确地预测出蛋白质的稳定性或者是热力学性质。在将来，会有 更好的“力场” （forcefields） ，蛋白质与水互作用也会有更好的模型，构象采样 也会有更快的方法，哪怕是使用现如今最好的电脑它们也受到很大的限制。

The early days saw hopes of finding simple sequence patterns—say of hydrophobic, polar, charged,and aromatic amino acids—that would predict protein structures and stabilities. That has not materialized. Nevertheless, the results of the detailed atomic simulations described above give optimism that atomically detailed modeling is systematically improving and is contributing to our understanding of protein sequence-structure relationships.

早期，我们曾看到寻找到简单序列模式的希望，即疏水，极性，带电和芳香氨基 酸，并想以此来预测蛋白质的结构和它的稳定性。那个希望却并没有成真。然而，上述详细的原子模拟结果仍能让我们觉得乐观，因为基于原子的详细模型得到系 统的改善，它也加深了我们对蛋白质-序列的关系的理解。

图2 Modern physical models can compute the folded structures of some small proteins. Using a high-performance custom computer called Anton (48), Shaw and co-workers observed reversible folding and unfolding in more than 400 events across 12 small proteins to structures within 4.5 ? of the experimental structure (15). The experimental structures are shown in red, and the computed structures are blue. Shown are the name, PDB identifier, and RMSD (root-mean-square deviation between alpha carbon atoms) between the predicted and experimental structures. [Adapted with permission (15)]

现代物理模型可以计算一些小型蛋白质的折叠结构。使用高性能自定义的安东计算机 (48), Shaw 和同事对 实验结构在 4.5 ? 内的 12 个蛋白质结构进行了 400 多次实验，观察到了蛋白质的可逆折叠和展开 4.5(15)。红色所示的实验结构,蓝色是计算结构。上面显示了预测结构与实验结构的名字，PDB 标识符, ＲＭＳＤ(α 碳原子之间的均方根偏差)。

The Rate Mechanism of Protein Folding
蛋白质折叠的速率机制

At a meeting in Italy in 1968, Cyrus Levinthal raised the question (17) of how, despite the huge number of conformations accessible to it, a protein molecule can fold to its one precisely defined native structure so quickly (microseconds, for some proteins). How does the protein―know‖ what conformations not to search?

1968 年在意大利举行的一个会议上，Cyrus Levinthal 提出了这个问题(17)，尽 管一个蛋白质有大量的构象，但是蛋白质是怎样快速折叠成它精确定义的天然结 构的（对于一些蛋白质，只要几微秒） 。蛋白质怎么知道什么样的构象是不用去搜索的？

This question led to a major experimental quest to characterize the kinetics of protein folding and to find folding intermediates, which are partially structured states along the―folding pathway‖(18, 19). The hope was that snapshots of the chain caught in the act of folding would give insights into folding ―mechanisms,‖the rules by which nature performs conformational searching.The experimental challenge was not just to measure atom-by-atom contacts within the heterogeneous interior of a protein molecule, but to do it on the fly, over microsecond-to-second time scales. This drove development of a powerful arsenal of new experimental methods, including mutational studies, hydrogen exchange, fluorescence labeling, laser temperature jumps, and single molecule methods [reviewed elsewhere (7)].

这个问题引发了一个重要的实验探索，以描述蛋白质折叠动力学的特征、寻找折 叠的中间产物，中间产物的部分结构化状态是沿着―折叠途径‖的所体现的状态 (18，19)。我们希望能够捕捉到链在折叠过程中的快照，希望这些快照能够让我 们洞察出折叠机制，而折叠机制就是天然态进行构象搜索的规则。这个实验的挑 战不仅仅是测量蛋白质分子异构内部中原子与原子的接触,还要在微秒的时间内 飞速的实现它。而这也推动了新实验方法的发展，这些实验方法包括突变研究, 氢交换、荧光标记、激光温度跳跃,和单分子方法[参考(7)]。

The general-principles solution of the needle-in-a-haystack conundrum emerged from polymer statistical thermodynamics. Studies of the chain entropies in models of foldable polymers showed that more compact, low-energy conformational ensembles have fewer conformations (12, 20–23), indicating that protein-folding energy landscapes are funnel-shaped (Fig. 3). Protein folding landscapes are narrower at the bottom; there are few low-energy, native-like conformations and many more open unfolded structures. A protein folds by taking random steps that are mostly incrementally downhill in energy. Steps need only be favorable by one to two times the thermal energy to reach the native structure rapidly (24). Insights from funnels, however, have not yet been sufficient to improve computer search methods. A landscape that appears smooth and funnel-shaped on a global scale can be rough on the local scales that are sampled in computer simulations.

在聚合物统计热力学中，人们找到那个如大海捞针般的难题的一般原则性解决方 法。对可折叠聚合物模型中链焓的研究表明，更紧密、能量更低的构象集合的构 象数量更少一些(12，20-23)，这表明蛋白质折叠能量图景是呈漏斗状的（如图 3 所示） 。蛋白质折叠图景底部更狭窄些; 低能类天然结构的构象很少，开放的未 折叠的结构很多。蛋白质折叠的步骤是随机的，并且是逐步沿着能量降低的方向 的。当能量是热能一到两倍的时，就能够很快的到达天然状态(24)。然而，从漏 斗图中得出的见解并不足以改进计算机搜索方法。图景看上去是光滑的，从全局 角度看是呈漏斗状的。图景也可以粗略的看成能在计算机模拟中进行采样的局部 结构。

But we are still missing a―folding mechanism.‖By mechanism, we mean a narrative that explains how the time evolution of a protein's folding to its native state derives from its amino acid sequence and solution conditions. A mechanism is more than just the sequences of events followed by any one given protein in experiments or in computed trajectories. We do not yet have in hand a general principle that is applicable to a broad range of proteins, that would explain differences and similarities of the folding routes and rates of different proteins in advance of the data, and that properly average, in some meaningful way, over ―irrelevant‖thermal motions.

但我们仍然缺乏一种―折叠的机制‖。机制，我们指的是这样的一个叙述，它解释 了蛋白质从氨基酸序列和它所处的溶液环境折叠回天然状态的时间演化。机制不 仅仅是在实验中或是在计算轨迹中给定一个蛋白质后，这个给定蛋白质的折叠顺 序。现在我们并没有这样一个原则：它适用于大部分蛋白质；在有数据之前，它 能够解释不同蛋白质折叠路径和折叠比例的差异性与相似性； 通过一些有意义的 方法，它能够适当地平均一下一些毫不相关的热运动。

One difficulty has been reconciling our ―macroscopic‖ understanding of kinetics (mass-action models) that result from ensemble-averaged experiments with our ―microscopic‖ understanding of the angstrom-by-angstrom changes of each protein conformation in computer simulations (energy landscapes).However, there are a few general conclusions(25). Proteins appear to fold in units of secondary structures. A protein's stability increases with its growing partial structure as it folds. And, a protein appears to first develop local structures in the chain (such as helices and turns) followed by growth into more global structures. Even though the folding process is blind, nevertheless it can be fast because native states can be reached by this divide-and-conquer, local-to-global process(26,27). Funneled landscapes predict that the different individual molecules of the same protein sequence may each follow microscopically different routes to the same native structure. Some paths will be more populated than others.

困难之一是协调我们对实验中动力学的宏观理解和计算机模拟中蛋白质构象埃 与埃之间变化的微观理解。然而，只有几个一般性的结论(25)。蛋白质似乎是在 单位二级结构中折叠。蛋白质的稳定性随着它折叠部分的增加而增加。并且，蛋 白质链的局部结构先开始变化(例如螺旋和轮)最后整个结构都开始变化。尽管折 叠过程是看不见的，然而它可以通过―分而治之‖和―局部与全局‖过程快速到达天 然状态。漏斗状的图景预测说，在用显微镜观察时，同一个蛋白质序列不同的单 个分子沿着不同的路径形成相同的天然结构。有些路径上的分子数比其他的路径 多。

图3 Proteins have a funnel-shaped energy landscape with many highenergy, unfolded structures and only a few low-energy, folded structures.Folding occurs via alternative microscopic trajectories.

蛋白质有一个漏斗状的能量图景，图景有许多高能未折叠的结构，和少量低能折叠结构。

Computing Protein Structures from Amino Acid Sequences
根据氨基酸序列计算蛋白质的结构

A grand challenge has been to develop a computer algorithm that can predict a protein's 3D native structure from its amino acid sequence. On the one hand, knowledge of native structures is a starting point for understanding biological mechanisms and for discovering drugs that can inhibit or activate those proteins. On the other hand, we know 1000-fold more sequences than structures, and this gap is growing because of developments in high-throughput sequencing. So, there is considerable value in methods that could accurately predict structures from sequences.

发展一种根据氨基酸序列就能预测蛋白质的结构的计算机算法是一个巨大的挑 战。一方面，了解天然结构的相关知识是了解生物机制、发现抑制或激活蛋白质的药物的一个出发点。另一方面，我们对结构的了解多过序列，而且由于高通量 测序技术的发展，这个差距在增大。因此，能根据序列准确预测其结构的方法是 有相当大的价值的。

Computer-based protein-structure prediction has been advanced by Moult and colleagues, in an event initiated in 1994 called CASP: Critical Assessment of protein Structure Prediction (28,29). Held every second summer, CASP is a community-wide blind competition in which typically more than 100 different ―target sequences‖ (of proteins whose structures are known but not yet publicly available) are made available to a community that numbers more than 150 research groups around the world. Each participating group applies some algorithmic scheme that aims to predict the 3D structures of these target proteins. After each CASP event, the true experimental structures are then revealed, group performances are evaluated, and community evaluations are published.

在 1994 发起的一个叫 CASP（蛋白质结构预测评估）的竞赛中，Moult 和他的 同事加快了基于计算机的蛋白质结构预测的进程。每两年的夏天举行一次 CASP，CASP 是一个全社会的无目的竞赛，来自全世界的 150 多个研究小组会得到 100 多个不同的目标序列(蛋白质，结构已知但并未向外界公开)。每个参与小组运用 一些算法方案来预测这些目标蛋白质的三维结构。每次 CASP 举行后，组委会随 后会公布用于实验中蛋白质真正的结构，评估所有小组的表现，出版整个评估结 果。

Currently, all successful structure-prediction algorithms are based on assuming that similar sequences lead to similar structures. These methods draw heavily on the PDB, which now contains more than 80,000 structures. However, many of these structures are similar, and the PDB contains only ~4000 structural families and 1200 folds (30).

当前，所有成功的结构预测算法都是基于相似序列有相似结构这个假设的。这些 方法主要是利用 PDB，现在 PDB 包含 80,000 多个结构。然而，这些结构中有很 多是相似的，并且 PDB 大约只包含 4000 个结构家系和 1200 折叠子(30)。

CASP-wide progress over the past 18 years is summarized in Fig. 4A. Prediction accuracies improved from CASP1 (1994) to CASP5 (2002) on the basis of several advances: (i) PDB expanded from ~1600 structures to 19,000 during that time. (ii) Better sequence search and alignment tools, such as Position-Specific Iterated Basic Local Alignment Search Tool (PSI-BLAST) (31), enabled the detection of more remote evolutionary relationships and more accurate sequence alignments. (iii) A strategy, called the ―fragment assembly approach‖ (32–35), was developed that can often improve predictions when a similar sequence cannot be found in the PDB.

CASP 在过去 18 年所取得的成绩都总结在图 4A 中。从 CASP1 (1994) 到 CASP5 (2002)，预测准确性得到提高是由于以下几方面的进步： (i)在那段时间，PDB 从 1600 个 结 构 扩 展 到 19,000 个 (ii) 更 好 的 序 列 搜 索 和 校 正 工 具 ，例 如 Position-Specific Iterated Basic Local Alignment Search Tool (PSI-BLAST) (31)，它 使得探测远亲进化关系和更准确的序列比对成为可能。(iii) ―片段拼接方法 ‖ (32-35)，当我们在 PDB 库中找不到相似序列时，利用这个方法也可以提高预测。

 If the target protein's sequence is related to a sequence that is already in the PDB, predicting its structure is usually easy (Fig. 4). In such cases, target protein structures are predicted by using―template-based modeling‖(also called homology modeling or comparative modeling). But when there is no protein in the PDB with a sequence resembling the target's, accurately predicting the structure of the target is much more difficult. These latter predictions are called―free modeling‖(also called ab initio or de novo prediction). One of the most successful free-modeling techniques is fragment assembly, described below. Many prediction methods are hybrids, combining template-based modeling, fragment assembly, and other strategies.

 如果目标蛋白质序列和 PDB 中已有序列有关，那么预测它的结构通常是较容易 的(图 4). 在这些情况下，通常使用―基于模板的模型‖ (也称同源模建或比较建模 法)来预测目标蛋白质。但是当 PDB 中没有与目标序列相似的蛋白质序列时，准 确预测目标蛋白质的结构就相当困难。之后预测方法叫做―自由建模‖ (也叫做从 头计算法或者是从头预测)。其中最成功的一个自由建模技术是片段拼接，叙述 如下。许多预测方法都是混合算法，结合了基于模板的建模方法、片段拼接还有 其他的方法。

 In fragment assembly (32–35), a target protein sequence is deconstructed into small, overlapping fragments. A search of the PDB is performed to identify known structures of similar fragment sequences, which are then assembled into a fulllength prediction. The qualities of fragments and their assemblies are assessed by using some form of scoring function that aims to select more native-like protein structures from among the many possible combinations. Problems of folding physics described above share more commonality with free modeling than with template-based modeling.

 在片段拼接中(32-35)，目标蛋白质序列被解构为小型重叠片段。PDB 搜索的任 务就是识别有相似片段的已知结构，然后形成一个全局的预测。我们用某种形式 的计分函数来评估片段的质量和它们的相似性，计分函数就是从许多个可能的组 合之中挑选出更为接近的蛋白质天然结构。相较于基于模板的建模，上述的折叠 现象与自由建模具有更多的共性。

 Since CASP6, although overall progress has slowed (Fig. 4A), there has been systematic, incremental progress (36). The best groups can now on average produce models that are better than the single best template from the PDB. Progress has been made toward successfully combining multiple templates into a single prediction. Substantial improvements have been observed for free-modeling targets shorter than 100 amino acids, although no single group yet consistently produces accurate models. Larger free-modeling targets remain challenging. Several recent algorithmic developments—to predict residue-residue contacts from sequence alone (37–39) and to more sensitively and accurately identify remote homologs (40)—promise to further improve prediction accuracy.

 CASP6 以来,尽管总体进展放缓(图 4A),但是仍有系统的、循序渐进的进步(36)。现在，表现最好的小组一般可以构建出比 PDB 中最好的模板还要好的模型。现 在已经成功的把多模板结合到单一预测中。尽管还没有单个的小组能持续的构建 出准确的模型，但是对于自由建模中氨基酸序列短于 100 的目标蛋白质，已经有 实际性的改善了。利用自由建模预测更大型的目标仍富有挑战性。最近发展的几 个算法――仅根据序列预测残基与残基的联系(37-39)，敏感而准确地辨认远亲同 源染色体(40)――有望进一步提高预测的准确性。

 The performance of two of the best fully automated server predictors during CASP9 (41)are shown in Fig. 4B: HHPred, a pure template-based modeling tool (42), and ROSETTA, a hybrid tool that combines fragment assembly and template-based modeling with all-atom refinement (43).For some fraction of CASP targets [~10%, based on a cutoff of 85 Global Distance Test–Total Score (GDT-TS) (44), which is defined in Fig. 4, legend], the best predictions are now accurate enough to interpret biological mechanisms, to guide biochemical studies, or to initiate a drug-discovery program (which requires structural errors of less than 2 to 3 ?). However, it remains a challenge to predict the other 90% of protein structures this accurately. In addition, it is critical to also improve physics-based technologies and to reduce our dependence on knowledge of existing structures, so that we can ultimately study proteinmotions, intrinsically disordered proteins,induced-fit binding of drugs, and membrane proteins and foldable polymers, for which databases are too limited.

 图 4B 展示了在 CASP9 期间两个全自动服务器预测的表现：HHPred----纯模板预 测工具 (42)和 ROSETTA----一个结合了片段拼接和基于模板的混合工具(43)。对 于一部分 CASP 目标蛋白质 ［~10%］ ，现在最好的预测能够准确的阐述生物机制，能够引导生物化学的研究，或者开启一个药品研发项目（要求结构误差小于 2~3 ?)。然而，用这个精度去预测其他的 90%的蛋白质的结构仍然是一个挑战。另 外，改进基于物理的技术、减少对已有结构的依赖性也是很重要的，因此我们要 在根本上研究蛋白质的运动、固有的无序蛋白、诱导结合的药物、膜蛋白和可折 叠的聚合物，因为数据库太有限了。

图4 Historical and present performance in CASP. Model quality is judged by using GDT-TS (44), which is approximately the percentage of residues that are located in the correct position. (A) Evolution of accuracy over the history of CASP, spanning 18 years. Each target is classified according to an approximate measure of difficulty that incorporates both the structural and sequence similarity to proteins of known structure (36). Each dot represents the best prediction (across all participants) for a given target. (B) Summary of prediction accuracy in CASP9 (41). We highlight the performance of two of the best automated server algorithms. Selected predictions are superimposed on the corresponding native structures to give a visual sense of the accuracy level that can be expected.

CASP 的历史表现和当前表现。用 GDT-TS 来判断模型质量 (44), GDT-TS 是指位于正确位置的残基的近似 百分比。(A)CASP18 年的准确性进化史。每个目标是根据相似的结构和序列合并到已知结构的难度来分类 的 (36)。每个点代表一个给定目标的最佳预测(在所有参与者中)。(B) CASP9 预测准确性总结(41)。我们突 出了两个全自动服务器算法的性能。所选择的预测叠加在对应的天然结构上是为了提供我们期待的视觉上 的准确性。

Protein Folding: The Legacy of a Basic Science
蛋白质折叠：基础科学的遗产

Protein folding is a quintessential basic science. There has been no specific commercial target, yet the collateral payoffs have been broad and deep. Specific technical advances are reviewed elsewhere(7); below, we describe a few general outgrowths.

蛋白质折叠是基础科学的精髓。没有特定的商业目标，并行的成果广泛而深入。具体的技术进步参考 (7),下面,我们描述几个一般性成果。

Growth of protein-structure databases.Today, more than 80,000 protein structures are known at atomic detail and publically available through the PDB. New structures are being added at a rapid pace, supported by the National Institutes of Health (NIH)–funded Protein Structure Initiative, which was developed in part to inform protein structure prediction.

蛋白质结构数据库的增长。今天,超过 80000 个蛋白质原子层面上的已知结构可 以通过 PDB 获得。由美国国立卫生研究院(NIH)支持——提供资金的蛋白质结构 计划,新结构正在快速的添加到数据库中,在某种程度上，这也是发展来通告蛋白 质结构的预测。

Advances in computing technology.Understanding protein folding was a key motivation for IBM's development of the Blue Gene supercomputer (45), now also used to study the brain, materials, weather patterns, and quantum and nuclear physics. Protein folding has also driven key advances in distributed-grid computing, such as in Folding@home, developed by Pande at Stanford, in which computer users all over the world donate their idle computer time to perform physical simulations of protein systems (46). Folding@home, which now has more than one million registered users and an average of 200,000 user-donated CPUs available at any one time, provided some of the earliest simulations showing that MD simulations can accurately predict folding rates (47). The Anton computer from DE Shaw Research, custom designed to simulate biomolecules, gives several orders of magnitude better performance than conventional computers (48). Advances in computer technology have led to major advances in forcefields and to more reliable atomic-level insights into biological mechanisms.

计算技术的进步。了解蛋白质折叠是 IBM 的―蓝色基因‖超级计算机发展(45)的重 要的动机,现在也用于研究大脑、材料、天气模式,和量子物理学和核能。蛋白质 折叠也推动了分布式计算的重要进步,比如由斯坦福大学潘德创立的 ―折叠之家‖ （Folding@home） ,它让世界各地的计算机用户捐出电脑的闲置时间来进行蛋白 质系统的物理模拟 (46)。―折叠之家‖（Folding@home）现在有超过一百万的注 册用户, 在任何时间有平均 200000 个用户捐出的 CPU 可以使用, ―折叠之家‖提供了一些最早的模拟，这些模拟表明分子动力学模拟可以准确预测折叠率(47)。DE Shaw Research 的安东计算机是用户定制的生物分子模拟计算机，它的性能比 传统的电脑好出几个数量级(48)。计算机技术的进步带来了力场的重大进步以及 对生物机制在更可靠的原子层面的理解。

Improvements in biomolecular force fields. Computer processing power has advanced at the Moore's law rate, doubling every ~2 years. But equally important, forcefields have kept pace. Increased computer power leads to longer computed time scales, which puts more stringent demands on the accuracies of biomolecular forcefields. In a pioneering paper in 1977, McCammonet al. showed that the BPTI protein was stable in computer simulations during a computed time of 10 ps (49). Today, small proteins are typically stable in explicit-water simulations for 5 to 8 orders of magnitude longer—microseconds to milliseconds of computed time (50). Achieving such advances has required continuous improvements in forcefield accuracy.

生物分子力场的改进。计算机处理能力以摩尔定律的速率增长，即大约每 2 年增 加一倍。但同样重要的是,力场也保持同样的速率增长。计算机能力的增加扩大 了计算时间的规模,这也对生物分子力场的精度提出了更严格的要求。在 1977 年 的一篇开创性论文中,McCammonet 表明,BPTI 蛋白质在计算时间为 10 ps 的计算 机模拟中是稳定(49)。现今, 在微秒到毫秒的计算时间的 5 到 8 个数量级内，小 蛋白质在溶剂模型中一般是稳定的 (50)。实现这种进步要求不断提升力场的精 度。

New sociological structures in the scientific enterprise. Protein folding has driven innovations in how science is done. CASP was among the first community-wide scientific Competitions /collaborations, a paradigm for how grand-challenge science can be advanced through an organized communal effort. Other such competitions have followed, including Critical Assessment of Prediction of Interactions (CAPRI) (predicting protein-protein docking) (51), SAMPL (predicting small-molecule solvation free energies, and ligand binding modes and affinities) (52), and GPCR-Dock ( predicting structures for G-protein coupled receptors, a pharmaceutically important category of membrane proteins) (53), among many others. Protein folding has also pioneered―citizen science,‖such as in Folding (46) and Robetta@home and in a computer game called Foldit (54), in which the public engages in protein folding on their home computers.

科技企业的新社会结构。蛋白质折叠推动着科学是如何实现这方面的创新。CASP 是最早覆盖全社会的科学竞赛/协作,是一个颇具挑战的科学通过有组织的集体努 力取得进步的范例。之后也有这样的竞赛,其中有一些是： 交互预测评估 （CAPRI） (预测蛋白质与蛋白质的对接)(51),SAMPL(预测小分子溶剂的自由能量,以及配体 结合的模式和亲疏性)(52)和 GPCR-Dock(预测 G-protein 耦合的受体的结构，这是 配药学中的重要膜蛋白种类 )(53) 。蛋白质折叠也是 ― 公众科学 ‖ 的先驱 , 如在 ―Folding‖(46)和―Robetta@home‖和一款叫做―Foldit‖的电脑游戏 (54),公众把蛋白 质折叠融入到他们的家用电脑中。

New materials: Sequence-specific foldable polymers.The principles and algorithms developed for protein folding have led to nonbiological, human-made proteins and to new types of polymeric materials. In particular, proteins have been designed that bind to and inhibit other proteins (fig. 5A) (55), have new folds (56), have new enzymatic activities (57), and act as potential new vaccines (58). Also, a class of nonbiological polymers has emerged, called―foldamers,‖that are intended to mimic protein structures and functions (59–62). Foldamers already have broad ranging applications (63–67) as inhibitors of protein-protein interactions, broad-spectrum antibiotics, lung surfactant mimics, optical storage materials, a zinc-finger–like binder, an RNA protein binding disrupter for application in muscular dystrophy, gene transfection agents, and―molecular paper‖(Fig. 5B). Although such materials have potential applications in biomedicine and materials science, they also provide a way for us to test and deepen our understanding of protein folding.

新材料： 特定序列的可折叠聚合物。为蛋白质折叠而发展的原理和算法产生了 非生物的人造蛋白质和新型聚合物。特别是，设计了一些能够捆绑和抑制其他蛋 白的蛋白质 (如图 5A) (55)、 有新折叠的蛋白质(56)、 有新的酶活性的蛋白质(57)，和一些能作为潜在的新疫苗的蛋白质(58)。并且，一个非生物的聚合物类已经出 现了，我们称之为―非天然折叠体‖，它旨在模仿蛋白质的结构和功能 (59-62)。―非 天然折叠体‖已经拥有广泛的应用(63-67)：作为蛋白质与蛋白质相互作用的抑制 剂、用途广泛的抗生素、肺表面活化剂仿造物、光存储材料、锌指结构黏合剂，应用于肌肉萎缩症 RNA 蛋白结合的破坏剂、基因转染载体和―molecular paper‖ (图 5B)。这些材料在生物医学和材料科学有潜在的应用，他们也为我们提供了一 种方法来测试和加深我们对蛋白质折叠的理解。

Protein-folding diseases.Protein-folding research began before it was known that there are diseases of protein folding. Before 1972, it was assumed that all infectious diseases were transmitted through the DNA and RNA carried by viruses and bacteria. But Prusiner's studies of a patient with Creutzfeldt-Jakob disease (CJD) led to a previously unrecognized disease mechanism—namely, protein misfolding (68). Protein misfolding is now be known to be important in many diseases, including CJD and type II diabetes, as well as neurodegenerative diseases such as Alzheimer's, Parkinson's, Huntington's, and amyotrophic lateral sclerosis. The protein-folding enterprise has provided important underpinnings for our understanding of folding diseases。

蛋白质折叠疾病。在知晓蛋白质折叠会带来疾病之前，蛋白质折叠的研究就 已经开始了。1972 年以前,人们认为传染病都是通过病毒和细菌所携带的 DNA 和 RNA 进行传染的。但 Prusiner 在研究一个克雅氏病(CJD)患者时发现了一种以 前未被识别的疾病机制，就是蛋白质错误折叠(68)。现在人们知道蛋白质错误折 叠，是因其在众多疾病中的重要性,这些疾病包括克雅氏病和 II 型糖尿病,以及一 些神经退行性疾病如阿尔茨海默氏症、帕金森氏症、 亨廷顿氏和肌萎缩性脊髓侧 索硬化症。蛋白质折叠企业对我们理解折叠疾病提供了重要的基础。

图5 Designed proteins and foldamers. (A) A protein inhibitor that was designed by computer to bind to hemagglutinin, an influenza protein. After design, the inhibitor was crystallized in a complex with hemagglutinin. The designed structure is in remarkably good agreement with experiment, particularly for the side chains involved in binding. [Adapted with permission (55)] (B) Peptoids are synthetic, foldable, protein-inspired polymers that have various applications. Shown here are that were designed as chains of alternating hydrophobic (gray) and either positively (blue) or negatively (red) charged side chains that spontaneously form a thin 2D structure called molecular paper. [Reprinted by permission (67)]

设计的蛋白质和非天然折叠体。(A)蛋白质抑制剂是由计算机设计来结合血凝素的，血凝素是一种流感蛋白 质。设计后, 抑制剂在一个有血凝素复合体中结晶。设计结构在实验中效果非常好，特别是对于结合中的 侧链。(经许可改编(55))(B) 拟肽是人造的,可折叠，蛋白质激励的，拥有各种应用的聚合物。这里显示的是 被设计成交替疏水 (灰色)和带正电荷(蓝色)或负电荷(红色)的侧链,它自发的形成于一个叫做“molecular paper”的稀疏的二维结构。(本文经许可转载(67))

Unsolved Problems of Protein Physical Science
蛋白质自然学科的未解问题

Is the protein-folding problem―solved‖yet (69)? We believe it is no longer useful to frame the question this way. Protein folding is now a whole field of research—a large, growing, and diverse enterprise. A field of science—such as physics, chemistry, or biology—is bigger than any individual research question. A field is self-perpetuating; a few old puzzles generate more new puzzles. For the field of protein physical science, the future is at least as compelling as the past. Here are some of the unsolved problems:

蛋白质折叠的难题都已经解决了吗?(69)我们认为这样解决问题不再有用。现 在蛋白质折叠是一个大型的,不断成长,多种多样的研究领域。它是一个包含了物 理、化学、生物的科学领域，它的问题比任何一个领域的研究问题都要宽泛。它 是一个自身能够永久存在的领域；几个旧问题能够产生更多的新问题。蛋白质自 然学科的未来至少会和它的过去一样引人入胜。这里有一些尚未解决的问题:

We have little experimental knowledge of protein-folding energy landscapes. We cannot consistently predict the structures of proteins to high accuracy. We do not have a quantitative microscopic understanding of the folding routes or transition states for arbitrary amino acid sequences. We cannot predict a protein's propensity to aggregate, which is important for aging and folding diseases. We do not have algorithms that accurately give the binding affinities of drugs and small molecules to proteins. We do not understand why a cellular proteome does not precipitate, because of the high density inside a cell. We know little about how folding diseases happen, or how to intervene. Despite their importance, we still know relatively little about the structure, function, and folding of membrane proteins (70, 71). We know little about the ensembles and functions of intrinsically disordered proteins(72), even though nearly half of all eukaryotic proteins contain large disordered regions. This is sometimes called the ―protein nonfolding problem‖ or ―unstructural biology.

我们没有蛋白质折叠能量图景的实验知识。我们不能持续地高精度地预测蛋白质的结构。对任意氨基酸序列，我们对折叠路径或过渡状态没有一个定量的微观理解。我们预测不到蛋白质的聚集倾向，聚集倾向对衰老和折叠疾病是非常重要的。我们没能够给出药品、小蛋白与蛋白质的结合能力的准确算法。我们不理解为什么在高密度的细胞内，细胞蛋白质组不沉淀。我们几乎不了解折叠疾病是如何发生的,或者该怎样干预它。尽管膜蛋白很重要,我们对它的结构、 功能和折叠的了解仍然相对较少(70、 71)。我们对内在的无序蛋白质的总体效果和功能知之甚少(72),哪怕有近一半的真 核蛋白质含有大量无序区域。这有时也被称为―蛋白非折叠问题‖或―无结构生物 学。

Summary
概要

Fifty years ago, the protein-folding problem was born as a grand challenge of basic science. Since then, our understanding has advanced considerably. And, outgrowths of protein folding include the commercial development of new computers, such as IBM's Blue Gene; new modes of citizen science, including Folding@Home and Foldit; the development of communal scientific competitions, such as CASP; a database of now more than 80,000 protein structures; the Moore's Law advancement in biomolecular simulation forcefields; new areas of materials science based on foldable polymers; and a foundation for understanding whole new classes of diseases—such as Alzheimer's, Parkinson's, and type II diabetes, called folding diseases—that were not even known when the protein-folding problem was first identified. In times when there are pressures on science budgets for immediate payoffs, it is worth repeating the well-worn point that untargeted basic science often pays off in unexpected ways.

五十年前,蛋白质折叠问题作为基础科学的巨大挑战而诞生。从那时起,我们的理解大幅度的加深。蛋白质折叠的产物，包括新电脑的商业开发,如 IBM 的蓝色 基因；公民科学的新模式，如―折叠之家‖（Folding@Home）和 ―蛋白质折叠‖ （Foldit） ；公共科学竞赛的发展,如 CASP；现已有超过 80000 个蛋白质结构的数 据库； 在生物分子模拟力场中摩尔定律发展； 材料科学基于可折叠聚合物的新领 域；理解新类型疾病的基础，如阿尔茨海默氏症,帕金森氏症,和 II 型糖尿病等一 些蛋白质折叠问题被认同之前我们都不知道的疾病。在科学预算肩负着及时回报的压力的时代，有必要重复这个老生常淡的观点： 无针对性的基础科学有时会以意想不到的方式来回报。
</div>

<script>
	var $=function(id){return document.getElementById(id)}
	var arr = $('dbLayout').innerHTML.split(/[\n|\r\n]+/), len=arr.length
	var str = '<table align="left" border="0" cellspacing="0" cellpadding="0" width=100% style="font:1em/1.5 Helvetica,Arial,\'Microsoft YaHei\',sans-serif;">';
	for(var i=1; i<len-1; i+=2) {
		str += '<tr><td align=left valign=top width=49%>'+arr[i]+'</td>'
				+  '<td align=left valign=top width=49%>'+arr[i+1]+'</td></tr>';
	}
	str += '</table>';
	$('dbLayout').innerHTML = str;
</script>
